editor/export/editor_export_platform.cpp

```diff
@@ -1234,6 +1234,25 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 			plugin->_end_customize_scenes();
 		}
 	}
+
+	// Postprocess
+	for (int i = 0; i < export_plugins.size(); i++) {
+		if (export_plugins[i]->get_script_instance()) { //script based
```

Comment:
Comments should start with upper case and end in a period, same above.

Reply:
I fIxed it.

But I need to remind you，this line just copied from other line. I'm not sure about we should fix other comments or not.

Reply:
What I've been told is that if the line is part of code that's modified you should change it, but otherwise leave it

Reply:
Thanks.